tests/Feature/MasjidProfileTest.php

```diff
@@ -46,4 +47,13 @@ public function admin_user_can_update_masjid_profile_data()
             'value' => 'https://maps.app.goo.gl/abcd',
         ]);
     }
+
+    /** @test */
+    public function masjid_name_based_masjid_profile_data()
+    {        
+        $masjidName = Setting::get('masjid_name', config('masjid.name'));
+        
```

Comment:
Mas @pandigresik minta tolong diperbaiki dulu ini PHP Lint nya failed (ada syntax yang tidak sesuai kaidah).

```suggestion
    {
        $masjidName = Setting::get('masjid_name', config('masjid.name'));

```

Lihat nya di sini mas:
https://github.com/buku-masjid/buku-masjid/actions/runs/8608671682/job/23591411951#step:4:39